### DISCUSSION TITLE
No documentation about livewire:discover command being removed

### DISCUSSION THREAD

**Discussion by jobara** [livewire/livewire#7164] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Run `php artisan livewire:discover` 
2. error saying "Command "livewire:discover" is not defined."

### Additional information

I couldn't find any information about `livewire:discover` being removed or why. It isn't mentioned in the upgrade guide either. Is it no longer needed or replaced by something else? In poking through the history it appears it was removed with https://github.com/livewire/livewire/pull/5789

It would be helpful to document the commands that have changed or were removed. 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by nizzam** [comment]
What is the answer to this question ??

**Reply by joshhanley** [reply, marked as answer]
@jobara and @nizzam it was removed as it's no longer needed. PRs to the docs are welcome if you want to add a note in the upgrade guide 🙂

**Comment by jobara** [comment]
It looks like a [note](https://livewire.laravel.com/docs/upgrading#discovery) was added to the docs in https://github.com/livewire/livewire/pull/7606

**Reply by joshhanley** [reply]
@jobara ah perfect! Thanks for checking.
